cfgov/ask_cfpb/scripts/migrate_spanish_short_answers.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import unicode_literals
+
+import os
+
+from django.contrib.auth.models import User
+
+from bs4 import BeautifulSoup as bs
+
+from ask_cfpb.models.pages import AnswerPage
+
+
+tags = ['em', 'i', 'strong', 'b']
+
+false_positives = [1637]
+
+
+def run():
+    answers = AnswerPage.objects.filter(
+        language='es',
+        redirect_to_page=None,
+        live=True
+    )
+    migration_user_pk = os.getenv('MIGRATION_USER_PK', 9999)
```

Comment:
Is there a user with this PK in the production database? If not, won't this leave these revisions in a bad state, e.g. when trying to view the revision list for these pages, will an exception be raised because the foreign key will be invalid?

Reply:
@chosak There isn't a user with that PK -- the plan is to run this script once in production with an actual user supplied (as we did with the [recent answer_to_answerpage migration](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/ask_cfpb/scripts/deprecated/answer_to_answerpage_migration.py#L26-L27)). I think revisions without a user will show up in the revisions list but can cause some behind-the-scenes issues, which is why we opted for this approach.

Reply:
If the code falls back to using the user PK 9999, which doesn't exist, then the user is None (because we call `first()` on an empty queryset) and the script proceeds. It's not the end of the world to push a revision with no user, but it's much better to provide one, and we intend to provide one when we run these scripts.

All of our migration scripts are for one-time use in production, but we run them a number of times in dev. This routine is so that we can quickly iterate in dev without chasing errors.